plugins/services/src/js/columns/ServicesTableMemColumn.tsx

```diff
@@ -1,19 +1,60 @@
 import * as React from "react";
-import { NumberCell } from "@dcos/ui-kit";
+import { Trans } from "@lingui/macro";
+import { NumberCell, Tooltip } from "@dcos/ui-kit";
 
 import Pod from "../structs/Pod";
 import Service from "../structs/Service";
 import ServiceTree from "../structs/ServiceTree";
 import Units from "#SRC/js/utils/Units";
 
-export const ServiceMem = React.memo(({ resource }: { resource: string }) => (
-  <NumberCell>
-    <span>{Units.formatResource("mem", resource)}</span>
-  </NumberCell>
-));
+export const ServiceMem = React.memo(
+  ({
+    resource,
+    limit
+  }: {
+    resource: string;
+    limit: number | string | null;
+    id: string;
+  }) => {
+    if (limit != null && limit !== 0) {
```

Comment:
what's the exact thing we need to do here?

do we want to handle empty strings? do we only bail out in case of an exact `null`?